### PR TITLE
11.0 mig web_listview_range_select

### PR DESCRIPTION
**Pull request by aldo774**
Migration of the web_listview_range_select module to v11

### CODE REVIEW

**Review by mstuttgart — APPROVED** [review]
It's working! Thanks :+1:

File: web_listview_range_select/README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+To use this module, you need to:
+
+#. click a record;
+#. hold shift and click another record;
+#. you can repeat this operation as many times as you want.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/162/10.0
```
Comment (tarteo) [inline web_listview_range_select/README.rst]: Please change this to 11.0

File: web_listview_range_select/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+{
+    'name': 'List Range Selection',
+    'summary': """
+        Enables selecting a range of records using the shift key
+    """,
+    'version': '11.0.1.0.0',
+    'category': 'Web',
+    'author': 'Onestein,Odoo Community Association (OCA)',
+    'website': 'http://www.onestein.eu',
```
Comment (tarteo) [inline web_listview_range_select/__manifest__.py]: Can you change the website to https://github.com/oca/web, please.

File: web_listview_range_select/README.rst [hunk 3]
```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (tarteo) [inline web_listview_range_select/README.rst]: Please, update the README to comply the new template.

**Review by tarteo — CHANGES_REQUESTED** [review]
Thanks for migrating. I tested it on Runbot and it works perfectly.

File: web_listview_range_select/README.rst (hunk 3 above)
Comment (aldo774) [reply web_listview_range_select/README.rst]: Changes made :+1:

**Comment by aldo774** [comment]
Changes maded :+1:

**Review by rodrigets — APPROVED** [review]
LGTM :+1:

**Review by rafaelbn — APPROVED** [review]
Tested in runbot :+1: 

Please @tarteo review you review :smile: Thanks!

**Review by tarteo — APPROVED** [review]

**Pull request merged**